api/controllers/userController.js

```diff
@@ -0,0 +1,17 @@
+const { userService } = require("../services");
+
+const signUp = async (req, res) => {
+  try {
+    const { email, password, name, phoneNumber } = req.body;
+
+    if (!email || !password || !name || !phoneNumber) return res.status(400).json({ message: "KEY_ERROR" });
```

Comment:
코드가 너무 길어지면 줄 바꿈 해주세요.
```suggestion
    if (!email || !password || !name || !phoneNumber) 
      return res.status(400).json({ message: "KEY_ERROR" });
```